### ISSUE TITLE
Example Input Files Not Working

### ISSUE THREAD

**Issue by jfreeze95** [bjing2016/alphaflow#3]
Great work on the latest version of the paper and thanks for putting this repo out. 
I was trying to test the basic inference you outlined using either the ESMFlow or AlphaFlow models and weights and ran into problems at every corner.  I'll detail my specific issues below but repos always get increased usage when authors provide at least one full example input line for inference, so if you provide that I'm sure it would help many people checking out your code. Thanks!

**Trying ESMFlow Model**

> mkdir output
> mkdir weights
> python predict.py --mode esmfold --input_csv splits/atlas_test.csv --weights weights/esmflow_md_distilled_202402.pt --samples 5 --outpdb output/

_Output_

> 2024-02-26 12:54:34,511 [---] [INFO] Loading the model
> 2024-02-26 12:55:16,878 [---] [INFO] Model has been loaded
> 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:25<00:00,  5.08s/it]
> Traceback (most recent call last):
>   File "/---/alphaflow/predict.py", line 132, in <module>
>     main()
>   File "/---/miniconda3/envs/AlphaFlow/lib/python3.9/site-packages/torch/autograd/grad_mode.py", line 27, in decorate_context
>     return func(*args, **kwargs)
>   File "/---/alphaflow/predict.py", line 126, in main
>     f.write(protein.prots_to_pdb(result))
>   File "/---/alphaflow/alphaflow/utils/protein.py", line 163, in prots_to_pdb
>     prot = to_pdb(prot)
>   File "/---/miniconda3/envs/AlphaFlow/lib/python3.9/site-packages/openfold/np/protein.py", line 341, in to_pdb
>     chain_index = prot.chain_index.astype(np.int32)
> AttributeError: 'NoneType' object has no attribute 'astype'

Tried with esmflow_pdb_base_202402.pt weights as well...same result.

**Trying AlphaFlow Model**
_Preparing the MSA_

> python -m scripts.mmseqs_query --split splits/atlas_test.csv --outdir output
> COMPLETE: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████| 450/450 [elapsed: 00:02 remaining: 00:00]
> 

SUCCESS!

_Running Inference_

> python predict.py --mode alphafold --input_csv splits/atlas_test.csv --msa_dir output/ --weights weights/alphaflow_pdb_distilled_202402.pt --samples 5 --outpdb output/
> 2024-02-26 13:17:56,383 [---] [INFO] Loading the model
> Traceback (most recent call last):
>   File "/---/alphaflow/predict.py", line 132, in <module>
>     main()
>   File "/---/miniconda3/envs/AlphaFlow/lib/python3.9/site-packages/torch/autograd/grad_mode.py", line 27, in decorate_context
>     return func(*args, **kwargs)
>   File "/---/alphaflow/predict.py", line 78, in main
>     model = model_class(**ckpt['hyper_parameters'], training=False)
>   File "/---/alphaflow/alphaflow/model/wrapper.py", line 496, in __init__
>     self.model = AlphaFold(config,
>   File "/---/alphaflow/alphaflow/model/alphafold.py", line 73, in __init__
>     self.extra_msa_stack = ExtraMSAStack(
> TypeError: __init__() missing 2 required positional arguments: 'opm_first' and 'fuse_projection_weights'

Thanks again for your assistance. Looking forward to trying out this great work.

**Comment by bjing2016** [comment]
Can you check if the OpenFold version is correct? A previous version of the README had the wrong install command --- see https://github.com/bjing2016/alphaflow/issues/2 where the issue sounds similar to what you describe with the ExtraMSAStack.

**Comment by jfreeze95** [comment]
Yep, that was it. Thanks!
In case anyone asks, the git request for the commit from OpenFold you specify won't work if you have Cuda 12.3, which is what led me to try the latest OpenFold commit which did work with 12.3 before I got Cuda 11.6 setup. 
Also, in mmseqs_query line 284 you have a hard-coded iloc[:3] which only allows the first three entries of a csv file to be processed. I ran into that when trying to use the atlas_test.csv file which will then throw an error during inference when it can't find the 4th entry's mmSeq folder.
